saplings/circuits/src/components/MainHeader.scss

```diff
@@ -48,4 +48,22 @@
       padding: 0.25rem;
     }
   }
+
+  .managementType {
+    color: var(--color-grey);
+    span {
+      color: var(--color-grey);
+      padding: 0.25rem;
+    }
+  }
```

Comment:
Rules for the class `managementType` are already set in the commit `Add circuit details page` f267a3f76f52e0aafd72980a3cc832106c5752fb . They are duplicated here

Reply:
Yeah, seems like some mistake with my commits.  I moved these to the `CircuitDetails.scss` file, so this is no longer relevant here.